prog/minio/minio_server_nexus.rb

```diff
@@ -171,16 +171,10 @@ def cluster
   end
 
   def available?
-    client = Minio::Client.new(
-      endpoint: minio_server.server_url,
-      access_key: minio_server.cluster.admin_user,
-      secret_key: minio_server.cluster.admin_password,
-      ssl_ca_file_data: minio_server.cluster.root_certs
-    )
-
-    server_data = JSON.parse(client.admin_info.body)["servers"].find { _1["endpoint"] == minio_server.endpoint }
+    server_data = JSON.parse(minio_server.client.admin_info.body)["servers"].find { _1["endpoint"] == minio_server.endpoint }
     server_data["state"] == "online" && server_data["drives"].all? { _1["state"] == "ok" }
-  rescue
+  rescue => e
+    Clog.emit("Minio server is down") { {minio_server: {minio_server_ubid: minio_server.ubid, error: e.message}} }
```

Comment:
https://github.com/ubicloud/ubicloud/pull/632

IMHO, `exception` isn't a property of `minio_server`. 
Here's a good example for Clog philosophy: https://github.com/ubicloud/ubicloud/blob/756b791f36d970455200697cd9fe78acbd787e49/bin/monitor#L25 
```
 { well_known_metadata_or_representation_name: { ... },
   going_concern: { a_field: "data", b_field: "more data" } }
```

If we think `minio_server` as `well_known_metadata_or_representation_name`
```suggestion
    Clog.emit("Minio server is down") { {minio_server: {ubid: minio_server.ubid}, unavailable_minio_server: {exception: Util.exception_to_hash(ex)}} }
```

If we think it as `going_concern`: `minio_server_down` or `unavailable_minio_server` etc.
```suggestion
    Clog.emit("Minio server is down") { {minio_server_down: {minio_server_ubid: minio_server.ubid, exception: Util.exception_to_hash(ex)}} }
```

Reply:
I think you don't even need minio_server_ prefix in your second example.

```
Clog.emit("Minio server is down") { {minio_server_down: {ubid: minio_server.ubid, exception: Util.exception_to_hash(ex)}} }
```

Reply:
Thanks guys, updated.